src/AppBundle/Resources/views/Report/Formatted/_report_summary.html.twig

```diff
@@ -0,0 +1,105 @@
+{% import 'AppBundle:Macros:macros-review.html.twig' as macros %}
+
+{# always true when sent as PDF at submittion time #}
+{% set readyToBalance = report.status.balanceState['state'] != 'not-started' %}
+{% set transOptions = {
+    '%moreless%': report.totalsOffset < 0 ? "less" : "more",
+} %}
+
+<div class="section" id="report-summary">
+    <h2 class="section-heading">Report summary</h2>
+
+    {% if report.hasSection('actions') %}
+        {{ macros.answerYesNo({
+            question: 'Significant Decisions',
```

Comment:
Could / should we be using a translation file for content?

Reply:
everything under src/AppBundle/Resources/views/Report/Formatted is very little translated. Not sure we want to plan to refactor that as in the future we'll send the data directly to Sirius and the PDF will die at a certain point